src/System.Private.ServiceModel/src/System/ServiceModel/Channels/AddressingVersion.cs

```diff
@@ -22,6 +22,7 @@ public sealed class AddressingVersion
         private string _defaultFaultAction;
         private const string AddressingNoneToStringFormat = "AddressingNone ({0})";
         private const string Addressing10ToStringFormat = "Addressing10 ({0})";
+        private const string Addressing200408ToStringFormat = "Addressing200408 ({0})";
```

Comment:
I am not sure on this one. Could not find the corresponding string on ReferenceSource :( Any ideas on where to find this one?